bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/dialog/Modal.java

```diff
@@ -130,8 +135,24 @@ protected void populateItem(ListItem<Component> item) {
 
         BootstrapResourcesBehavior.addTo(this);
     }
-
+    
     /**
+     * Factory method for the Modal's header 'closebutton'
+     * By default it generates the markup for original Bootstrap Modal closebutton
+     * 
+     * @param id The component id
+     * @return the close button
+     */
+    
+    protected Component createHeaderCloseButton(String id) {
+
+    	BootstrapButton btn = new BootstrapButton(id, Buttons.Type.Link);
+    	btn.setLabel(new  Model<String>("\u00D7"));
+    	btn.add(new AttributeModifier("class","close"));
```

Comment:
There is no need of this AttributeModifier. The class is already in the markup. Same for the label model.